tests/middleware/test_wsgi.py

```
@@ -0,0 +1,129 @@
+import io
```

Ruff (0.11.9)

`io` imported but unused

Remove unused import: `io`

(F401)